galaxy/worker/importers/base.py

```diff
@@ -63,10 +64,12 @@ def make_content(self):
             original_name = self.data.original_name
 
         # Check name
-        if not re.match(r'^[a-zA-Z0-9_-]+$', name):
+        if not re.match(constants.NAME_REGEXP, name):
```

Comment:
This is for traditional style roles import via github Repository()s right?

I don't think I'd enforce it for trad roles (yet?). Maybe a message or note about 
how the role name may not be compatible with collections, but let it import?

Reply:
Or is this used by both trad roles and roles-in-a-collection?

Reply:
This module is only executed for a traditional repository role. This change to the repository role regex was removed from this PR.